docs/en/final-task.md

```diff
@@ -2,15 +2,15 @@
 
 ## FAQ
 ### Question: Who makes assessment of the course project?
-Answer: Our mentor does verification of intermediate steps. The final assessment makes another randomly chosen mentor.
+Answer: Our mentor does verification at the intermediate steps. The final assessment makes another randomly chosen mentor.
 
-### Question: Does course project assessment being made as a review of regular task?
+### Question: Does the course project assessment being made as a review of regular task?
 Answer: Yes, in a dialogue format. Mentor who makes review leaves comments and can ask to fix something or explain.
 
 ### Question: Is there any additional requirements for commits?
 Answer: The final solution of the project should keep the history of commits being made during intermediate steps.
 
 ### Question: what to do if I can't get the course project done on the deadline/ going to overdue it for a couple days?
 Answer: You should notify the mentor who have been designated to the task review. Algorithm is simple:
-1. Create a Pull Request in any case
+1. Create Pull Request in any case
 2. Describe in detail your situation in Pull Request text (it's preferably at start and highlight it visually). If necessary leave your contacts.
```

Comment:
Describe in detail your situation in Pull Request text (it's preferable to include it at the beginning and to visually highlight it). Leave your contacts if it is necessary.